src/ElasticSearch/Aggregation/NullAggregationTransformer.php

```diff
@@ -4,21 +4,25 @@
 
 namespace CultuurNet\UDB3\Search\ElasticSearch\Aggregation;
 
+use CultuurNet\UDB3\Search\Facet\FacetTreeInterface;
+use LogicException;
+
 final class NullAggregationTransformer implements AggregationTransformerInterface
 {
     /**
      * @inheritdoc
      */
-    public function supports(Aggregation $aggregation)
+    public function supports(Aggregation $aggregation): bool
     {
         return false;
     }
 
     /**
      * @inheritdoc
+     * @never-return
```

Comment:
```suggestion
     * @return never
```
Could be a bit more consistent